packages/c/CMakePresets.json

```diff
@@ -50,6 +50,81 @@
       "name": "strict-ninja",
       "inherits": ["strict"],
       "generator": "Ninja"
+    },
+    {
+      "name": "release-static",
+      "binaryDir": "${sourceDir}/build/release-static",
+      "cacheVariables": {
+        "BUILD_SHARED_LIBS": "OFF",
+        "CMAKE_BUILD_TYPE": "Release",
+        "CMAKE_C_FLAGS": "-std=c99 -Wno-error -Wextra -Werror-implicit-function-declaration",
```

Comment:
Since we're defining C_FLAGS here we need to migrate them from the Makefile that's previously specified them. (Is there a way still to pass in additional C_FLAGS?)

Right now my OpenWrt Makefile has: `-fPIC -fhonour-copts -Wno-calloc-transposed-args -Wno-error -pthread -lrt`

Reply:
I'm not sure we need all of those flags, but the first two have been added for OpenWrt specific build issues.

Reply:
Also, is openwrt using any cmake to compile this? If so, which distribution, source or static?

Reply:
OpenWrt is using CMake. The git repo just comes with Makefiles that pull down the source, whilst the SDKs are a little more canned. It's with the latter that I've run into trouble with CMake binaries that won't fetch from GitHub.